libs/cart/src/drivers/interfaces/cart-item-service.interface.ts

```diff
@@ -0,0 +1,48 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { DaffCartItem } from '../../models/cart-item';
+import { DaffCart } from '../../models/cart';
+import { DaffCartItemInput } from '../../models/cart-item-input';
+
+/**
+ * The interface responsible for managing the items of a cart.
+ */
+export interface DaffCartItemServiceInterface<
+	T extends DaffCartItem,
+	U extends DaffCartItemInput,
+	V extends DaffCart
+> {
+	/**
+	 * List all of the available items of a cart
+	 */
+	list(cartId: DaffCart['id']): Observable<T[]>;
```

Comment:
It might not _really_ matter, but shouldn't this be `V['id']`?

Reply:
fixed in 822f0063